about-us.html

```diff
@@ -0,0 +1,543 @@
+                <img src="./img/" alt="connect_icon" class="header-connect-btn-img">>
+                connect
+            </button>
+        </div>
+    </header>
+    <main>
+        <section class="about-us-hero">
+            <div class="container">
+                <h1 class="about-us-title">ABOUT US</h1>
+                <nav class="about-us-navigation">
+                    <a href="./index.html">HOME</a>
+                    <a href="./about-us.html">ABOUT US</a>
+                </nav>
+            </div>
+        </section>
+        <section class="about-us">
+            <div class="container">
+                <h2 class="about-us-title">THE STORY</h2>
+                <p class="about-us-text">
+                    Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occae cat cupidatat non proident, sunt in culpa qui officia dese runt mollit anim id est laborum velit esse cillum dolore eu fugiat nulla pariatu epteur sint occaecat
+                </p>
+                <dl class="about-us-info">
+                    <dt class="about-us-info-subtitle">
+                        Total Iteam
+                    </dt>
+                    <dd class="about-us-info-description">
+                        2240+
+                    </dd>
+                    <dt class="about-us-info-subtitle">
+                        Profiles whitelisted
+                    </dt>
+                    <dd class="about-us-info-description">
+                        1000+                        
+                    </dd>
+                </dl>
+                <ol class="about-us-list">
+                    <li class="about-us-item">
+                        <img src="./img/" alt="icon_one">
+                        <h3>HIGH QUAILITY</h3>
+                        <p>Lorem Ipsum is simply dummy text of the printing and Ipsum typesetting industry. </p>
+                    </li>
+                    <li class="about-us-item">
+                        <img src="./img/" alt="icon_two">
+                        <h3>BIG COLLECTION</h3>
+                        <p>Lorem Ipsum is simply dummy text of the printing and Ipsum typesetting industry.</p>
+                    </li>
+                    <li class="about-us-item">
+                        <img src="./img/" alt="icon_three">
+                        <h3>TOP RESOURCE</h3>
+                        <p>Lorem Ipsum is simply dummy text of the printing and Ipsum typesetting industry. </p>
+                    </li>
+                    <li class="about-us-item">
+                        <img src="./img/" alt="icon_four">
+                        <h3>BIG COMMUNITY</h3>
+                        <p>Lorem Ipsum is simply dummy text of the printing and Ipsum typesetting industry. </p>
+                    </li>
+                </ol>
+            </div>
+        </section>
+        <section class="our-team">
+            <div>
```

Comment:
This element is an unnecessary wrapper. It can be removed without affecting the layout. Avoid unnecessary elements in the markup.